app/models/mongo_config.rb

```diff
@@ -0,0 +1,86 @@
+# Class so that funds can set their own dropdown lists of things
+# Marked for deletion after migration
+class MongoConfig
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include Mongoid::Userstamp
+  include Mongoid::History::Trackable
+  extend Enumerize
+
+  store_in collection: 'configs'
```

Comment:
This is the only change from the original file, right?

Reply:
Yep, that's right. Necessary to still get the model to point at the `configs` collection (rather than inventing one called mongo_configs`.